client/blocks/reader-post-card/style.scss

```diff
@@ -700,3 +700,19 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 .is-section-devdocs .reader-post-card__featured-image {
 	bottom: 16px;
 }
+
+.reader-post-card__discover-post-flag {
```

Comment:
The styling here is the same as the [recommended posts](https://github.com/Automattic/wp-calypso/blob/master/client/reader/stream/_style.scss#L417) but positioning is different. I opted to duplicate the rules here but am open to DRY'n it up.